docs/cli.html

```diff
@@ -754,13 +754,12 @@ <h2 id="invite">invite</h2>
   <div class="row-height">
     <div class="col-md-5 col-sm-12 col-height  docs-primary">
       <blockquote>
-<p>Manage the invitees/testers on your project. Can optionally specify a version or --remove.</p>
-</blockquote><p>  <strong>Usage:</strong> <code>zapier invite [user@example.com] [1.0.0]</code></p><p>Invite any user registered on Zapier to test your app. Commonly, this is useful for teammates, contractors, or other team members who might want to test, QA, or view your app versions. If you&apos;d like to provide full admin access, try <code>zapier collaborate</code>.</p><blockquote>
+<p>Manage the invitees/testers on your project. Can optionally --remove.</p>
```

Comment:
@BrunoBernardino it looks like documentation on invite versions were added without committing the change to `README-source.md`

Reply:
Found it - since CLI docs are generated from compiled `src` the `npm run build` should always be ran before `npm run docs`. Changed that in `package.json`.